### ISSUE TITLE
Binary file download

### ISSUE THREAD

**Issue by padraicc** [wdecoster/STRdust#3]
Hi Wouter,

There doesn't seem to be any read-to-use binary available for download under releases.

Kind regards,

Pádraic

**Comment by wdecoster** [comment]
Yes, that is also stopping the submission to bioconda at the moment. I usually build binaries for users with GitHub actions but that runs into a problem I don't yet know how to fix. I am attaching a musl binary to this answer. Any feedback is highly appreciated!

[STRdust.zip](https://github.com/wdecoster/STRdust/files/15470029/STRdust.zip)

**Comment by padraicc** [comment]
That's great. Thanks!
